src/random/array.ts

```diff
@@ -0,0 +1,14 @@
+import type {ArrayType} from '../type/classes/ArrayType';
+
+export function randomArray(type: ArrayType<any>): unknown[] {
```

Comment:
- Put all individual type random generator functions into a single module.
- Use *ArrowFunction*: `export const arr = () => ...`
- Use short names for exports, such as "arr", instead of "randomArr".